site/en/docs/extensions/reference/storage/index.md

```diff
@@ -13,76 +13,65 @@ The Storage API provides an extension-specific way to persist user data and stat
+
 {% Details %}
 {% DetailsSummary %}
 Can extensions use web storage APIs?
 {% endDetailsSummary %}
 
-Even though extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+While extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), we don't recommend it for the following reasons:
 
-- Extension's service worker cannot access `Storage`.
+- Extension's service worker can't access `Storage`.
 - Content scripts share storage with the host page.
 - Data saved using the `Storage` interface is lost when the user clears their browsing history.
 
 To move data from web storage APIs to extension storage APIs from a service worker:
 
 1. Create an offscreen document with a conversion routine and an [`onMessage`][on-message] handler.
-1. Add a conversion routine to an offscreen document.
-1. In the extension service worker check `chrome.storage` for your data.
-1. If your data isn't found, [create][create-offscreen] an offscreen document and call [`sendMessage()`][send-message] to start the conversion routine.
-1. Inside the offscreen document's `onMessage` handler, call the conversion routine.
+2. Add a conversion routine to an offscreen document. <!--Is 2 an alternative to 1? If so, I'd like to combine them and present them as options.-->
+3. In the extension service worker, check `chrome.storage` for your data.
+4. If your data isn't found, [create][create-offscreen] an offscreen document and call [`sendMessage()`][send-message] to start the conversion routine.
+5. Inside the offscreen document's `onMessage` handler, call the conversion routine.
 
-There are also some nuances with how web storage APIs work in extensions. Learn more in the
+There are also some nuances to how web storage APIs work in extensions. Learn more in the
 [Storage and Cookies][storage-and-cookies] article.
 
 {% endDetails %}
 
 ### Storage areas
 
-The Storage API is divided into the following four buckets ("storage areas"):
+The Storage API is divided into the following *storage areas*:
 
 [`storage.local`][prop-local]
-: Data is stored locally, which is cleared when the extension is removed. The quota limitation is approximately 10 MB, but can be increased by requesting the `"unlimitedStorage"` permission. Consider using it to store larger amounts of data.
-
-{% Aside 'caution' %}
-Before Chrome 114, the quota was approximately 5 MB.
-{% endAside %}
+: Data is stored locally and cleared when the extension is removed. The storage limit is 10 MB (5&nbsp;MB in Chrome 113 and earlier), but can be increased by requesting the `"unlimitedStorage"` permission. We recommend using `storage.local` using it to store larger amounts of data.
+<--I think we removed unlimitedStorage from the permissions list. How does that affect this?-->
```

Comment:
```suggestion
```

Reply:
I messed up when I have you remove `unlimitedStorage`. I just checked [where it's specified](https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/api/_permission_features.json;l=654). Can you please add it back.

Reply:
Done.